integration/src/test/resources/until-successful-test.xml

```diff
@@ -125,4 +164,19 @@
             </on-error-continue>
         </error-handler>
     </flow>
+
+    <flow name="nestedUntilSuccessfulScopes">
+        <until-successful maxRetries="1" millisBetweenRetries="10">
+            <test:invocation-counter name="outerScope"/>
```

Comment:
i think it would be better to use the test queue component instead of this counter

Reply:
Just tried using the `test:queue` instead of the counter. It doesn't work for me, since the only methods exposed by it are a `read` and a `write` to the queue. I need just to examine the number of times the processor is executed.